spec.html

```diff
@@ -689,7 +720,7 @@ <h1>DecorateConstructor ( _elements_, _decorators_ )</h1>
             1. Append _elementsAndFinisher_.[[Finisher]] to _finishers_.
           1. If _elementsAndFinisher_.[[Elements]] is not *undefined*,
             1. Set _elements_ to the concatenation of _elementsAndFinisher_.[[Elements]] and _privateElements_.
-            1. If there are two class elements _a_ and _b_ in _elements_ such that _a_.[[Key]] is _b_.[[Key]] and _a_.[[Placement]] is _b_.[[Placement]], throw a *TypeError* exception.
+            1. If there are two class elements _a_ and _b_ in _elements_ such that _a_.[[Kind]] is not `"initializer"` and _b_.[[Kind]] is not `"initializer"`, _a_.[[Key]] is _b_.[[Key]], and _a_.[[Placement]] is _b_.[[Placement]], throw a *TypeError* exception.
```

Comment:
Can this be made more readable? (I don't know if it valid from a sprc-grammar point of view)

```
1. If there are two class elements _a_ and _b_ in _elements_ such that all the following conditions are true:
  1. _a_.[[Kind]] is not `"initializer"`
  1. _b_.[[Kind]] is not `"initializer"`
  1. _a_.[[Key]] is _b_.[[Key]]
  1. _a_.[[Placement]] is _b_.[[Placement]]
1. Then throw a *TypeError* exception.
```

Reply:
Good idea, done